app/ts/components/TokenManager/index.tsx

```diff
@@ -113,7 +113,9 @@ const TokenCard = ({ token, onSelect }: TokenCardProps) => {
 	return (
 		<div class='aspect-square bg-neutral-800 p-4 flex items-end relative group outline-none' tabIndex={1}>
 			<div class='relative'>
-				<img class='w-12 h-12 mb-2' src={`/img/${token.address.toLowerCase()}.svg`} />
+				<object class='w-12 h-12 mb-2' data={`/img/${token.address.toLowerCase()}.svg`} type="image/png">
+					<img class='w-12 h-12' src="/img/ethereum.svg" />
+				</object>
```

Comment:
How does this result in a fallback image?

Reply:
an object element can load an external resource, just like how a video behaves, and will fallback to the underlying `<img>` while it fetches or if it failed to fetch the resource.